src/Mintette/MintetteOptions.hs

```diff
@@ -50,8 +60,31 @@ commandParser defaultSKPath =
         (command "serve" (info serveOpts (progDesc "Serve users and others")) <>
          command
              "dump-statistics"
-             (info (pure DumpStatistics) (progDesc "Dump statistics")))
+             (info (pure DumpStatistics) (progDesc "Dump statistics")) <>
+         command
+             "create-permission-keypair"
+             -- TODO: option to override the generated keypair location
```

Comment:
Ok, probably this TODO is about my previous comment (line 61 in `Main.hs`).

Also I think there is another TODO: we have to be more careful about rewriting existing key. Now we don't check that file with this name already exists when we write key, and user may accidentally delete their keys. It's unlikely to happen, but loosing secret key may be very expensive, so I think it makes sense to check whether file already exists and not rewrite it unless some flag is passed.